index.html

```diff
@@ -10,7 +10,12 @@
 </head>
 <body>
 
-    <h1>Hello world!!!</h1>
+    <div class="container">
```

Comment:
after container part, you need to add below "<h1 class="title">Weather App</h1>"